fluss-kafka/src/main/java/com/alibaba/fluss/kafka/KafkaRequestHandler.java

```diff
@@ -52,7 +57,34 @@ protected void handleInactive(KafkaRequest request) {
     }
 
     @Override
-    protected void handleApiVersionsRequest(KafkaRequest request) {}
+    protected void handleApiVersionsRequest(KafkaRequest request) {
+        short apiVersion = request.apiVersion();
+        if (!ApiKeys.API_VERSIONS.isVersionSupported(apiVersion)) {
+            request.fail(Errors.UNSUPPORTED_VERSION.exception());
+            return;
+        }
+        ApiVersionsResponseData data = new ApiVersionsResponseData();
+        for (ApiKeys apiKey : ApiKeys.values()) {
+            if (apiKey.minRequiredInterBrokerMagic <= RecordBatch.CURRENT_MAGIC_VALUE) {
+                ApiVersionsResponseData.ApiVersion apiVersionData =
+                        new ApiVersionsResponseData.ApiVersion()
+                                .setApiKey(apiKey.id)
+                                .setMinVersion(apiKey.oldestVersion())
+                                .setMaxVersion(apiKey.latestVersion());
+                if (apiKey.equals(ApiKeys.METADATA)) {
+                    // Not support TopicId
```

Comment:
Fluss uses `tableId`, a globally unique identifier of type `long`, to represent a table. Would it be feasible to use `tableId` as the `topicId` for Kafka in the future? For instance, by converting the `long` value into the `leastSignificantBits` and using a constant value for the `mostSignificantBits`.